KubeArmor/BPF/system_monitor.c

```diff
@@ -157,7 +163,7 @@ static __always_inline u32 get_task_ns_ppid(struct task_struct *task)
 {
     unsigned int level = task->real_parent->nsproxy->pid_ns_for_children->level;
 
-#if LINUX_VERSION_CODE < KERNEL_VERSION(4, 19, 0)
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4, 19, 0) && !defined(RHEL_RELEASE_GT_8_0)
```

Comment:
why don't we directly use "RHEL_RELEASE_CODE < RHEL_RELEASE_VERSION(8, 0)"?

Reply:
Not sure how could we club RHEL < 8 and this if condition.
Also it will error out on non RHEL systems if we directly use the condition, we need #ifdef (if that's what you wanted to infer )